README.md

```diff
@@ -14,12 +14,12 @@ There is support for both the AIR & SKY devices and the TEMPEST device.
 
 1. [Installation](#installation)
 2. [Docker Setup](#docker-setup)
-    1. [Docker Volume](#docker-volume)
-    2. [Docker Environment Variables](#docker-environment-variables)
+   1. [Docker Volume](#docker-volume)
+   2. [Docker Environment Variables](#docker-environment-variables)
 3. [Available Sensors](#available-sensors)
-    1. [Sensor Structure](#sensor-structure)
-    2. [High and Low Values](#high-and-low-values)
-    3. [Long Term Statistics](#Home-Assistant-Long-Term-Statistics)
+   1. [Sensor Structure](#sensor-structure)
+   2. [High and Low Values](#high-and-low-values)
+   3. [Long Term Statistics](#Home-Assistant-Long-Term-Statistics)
```

Comment:
My markdown formatter is apparently setup quite a bit differently than yours is. We probably need to upload a file that tells various editors how to format this (and other file types) so we have consistency when saving.

Reply:
I have not done anything specific on that - I use Visual Studio code on a Mac, to do everything. But whatever you can think of to align here, I will implement.

Reply:
Sweet, that's what I use as well. I'll see what I can add to the settings or files to give everyone the same output when saving in another PR